spec/acceptance/agent_spec.rb

```diff
@@ -4,15 +4,22 @@
   if version == '2.4' && default[:platform] =~ %r{(ubuntu-16.04-amd64|debian-9-amd64)}
     next
   end
+  # Zabbix supports only Debian 10 on Zabbix 4
```

Comment:
This comment seems worded the wrong way round.  Should be 'Debian 10 only supports zabbix 4'?  What about 4.2 and 4.4?